valhalla/sif/edgelabel.h

```diff
@@ -12,6 +12,12 @@
 namespace valhalla {
 namespace sif {
 
+constexpr uint32_t kInitialEdgeLabelCount = 1000000;
+constexpr uint32_t kInitialEdgeLabelCountMatrix = 4000000;
+constexpr uint32_t kInitialEdgeLabelCountBD = 500000;
+constexpr uint32_t kInitialEdgeLabelCountBDMatrix = 2000000;
+constexpr uint32_t kInitialEdgeLabelCountMM = 200000;
```

Comment:
gather them all here

Reply:
having them all here is great but for the ones that we changed we shouldnt lower them with this pr just for the sake of avoiding performance issues hidden with the change. its ok to increase some but especially bidir astar we should set back to its higher limit

Reply:
done